SA3/qa_web/views/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .add_content import *
+from .auth import *
+from .home import *
+from .profile import *
+from .question_operations import *
```

Comment:
Pylint gives a warning for wildcard imports here, but it is something we can overlook, as we really want everything from each sub-views.